.travis/runblack.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Run black (the Python formatter). The first script argument is a path to
```

Comment:
Can you please add the reason why we need this wrapper to the comment?

Reply:
Done in #145.